static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
+    updateProfile(clone, validator);
+  }
+});
+
+export function boundTelephoneInput(keySet: string[], countryKeySet: string[]): React$Element<*> {
+  const {
+    profile,
+    errors,
+    updateProfile,
+    validator,
+    updateProfileValidation,
+    updateValidationVisibility,
+  } = this.props;
+
+  let onBlur = () => {
+    updateValidationVisibility(keySet);
+    updateProfileValidation(profile, validator);
+    sendFormFieldEvent(keySet);
+  };
+
+  let currentCountry = R.toLower(R.defaultTo("", _.get(profile,  ['country'])));
+  return (
+    <div className={`bound-telephone ${validationErrorSelector(errors, keySet)}`}>
+      <ReactTelInput
+        defaultCountry={currentCountry}
+        flagsImagePath='/static/images/flags.png'
```

Comment:
It wouldn't, we still need to do the work of telling the JS side what our CDN is and attaching it as a prefix #1941